content/concept-amendments.md

```diff
@@ -86,13 +86,122 @@ When an amendment gets enabled for the network after the voting process, servers
 
 If your server is amendment blocked, you must [upgrade to a new version](tutorial-rippled-setup.html#updating-rippled) to sync with the network.
 
 
+#### How to Tell If Your `rippled` Server is Amendment Blocked
+
+***TODO: Question: Okay location for this doc? It is building upon a concept, but is tutorial-ish.***
+
+One of the first signs that your `rippled` server is amendment blocked is that an `amendmentBlocked` error is returned when you submit a transaction. Here's an example `amendmentBlocked` error:
+
+```
+{
+   "result":{
+      "error":"amendmentBlocked",
+      "error_code":14,
+      "error_message":"Amendment blocked, need upgrade.",
+      "request":{
+         "command":"submit",
+         "tx_blob":"479H0KQ4LUUXIHL48WCVN0C9VD7HWSX0MG1UPYNXK6PI9HLGBU2U10K3HPFJSROFEG5VD749WDPHWSHXXO72BOSY2G8TWUDOJNLRTR9LTT8PSOB9NNZ485EY2RD9D80FLDFRBVMP1RKMELILD7I922D6TBCAZK30CSV6KDEDUMYABE0XB9EH8C4LE98LMU91I9ZV2APETJD4AYFEN0VNMIT1XQ122Y2OOXO45GJ737HHM5XX88RY7CXHVWJ5JJ7NYW6T1EEBW9UE0NLB2497YBP9V1XVAEK8JJYVRVW0L03ZDXFY8BBHP6UBU7ZNR0JU9GJQPNHG0DK86S4LLYDN0BTCF4KWV2J4DEB6DAX4BDLNPT87MM75G70DFE9W0R6HRNWCH0X075WHAXPSH7S3CSNXPPA6PDO6UA1RCCZOVZ99H7968Q37HACMD8EZ8SU81V4KNRXM46N520S4FVZNSJHA"
+      },
+      "status":"error"
+   }
+}
+```
+
+***TODO: Is this a universal error? Should it be added here? https://ripple.com/build/rippled-apis/#universal-errors***
+
+You can verify that your `rippled` server is amendment blocked using the [`server_info`](reference-rippled.html#server-info) command. In the response, look for `result.info.amendment_blocked`. If `amendment_blocked` is set to `true`, your server is amendment blocked. For example:
+
+```
+{
+  "result": {
+    "info": {
+      "amendment_blocked": true,
+      "build_version": "0.90.0-b2",
+      "complete_ledgers": "6015234-6022198,6022744-6025498,6026905-6029106,6030549-6031191,6031472-6032443,6033090-6033095,6033339-6033537,6034791-6034950,6034980-6035798,6036466,6036516-6036703,6037478-6037671,6038085-6040415,6042691-6046229,6046413-6053861",
+...
+      "validation_quorum": 4,
+      "validator_list_expires": "2018-Jan-24 00:00:00"
+    },
+    "status": "success"
+  }
+}
+```
+
+If your server is not amendment blocked, the `amendment_blocked` field is not returned in the response.
+
+***TODO: Question: Update the `server_info` doc to include an `amendment_blocked` field descr? https://ripple.com/build/rippled-apis/#server-info. Does this amendment_blocked also display in `server_state`? https://ripple.com/build/rippled-apis/#server-state. If yes, also add descr there?***
```

Comment:
Respectively:

1. Yes.
2. Yes. (Confirmed [in the code](https://github.com/ripple/rippled/blob/3e9e0c4b53aa29775b02f5a092c79587cebef8d0/src/ripple/app/misc/NetworkOPs.cpp#L2141-L2142)—server_info and server_state use the same underlying function and there's no special code to change how this field displays.)
3. Yes.